jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/MaterialIconTests.java

```diff
@@ -1,25 +1,71 @@
 package io.github.epam.material.tests.displaydata;
 
-import static io.github.com.StaticSite.displayDataDefaultMaterialIconPage;
-import static io.github.com.StaticSite.displayDataLargeMaterialIconPage;
-import static io.github.com.StaticSite.materialIconFrame;
+import static io.github.com.StaticSite.materialIconPage;
+import static io.github.com.pages.displaydata.MaterialIconPage.*;
+import static io.github.com.MaterialNavigator.openSection;
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
 
 import io.github.epam.TestsInit;
+import org.hamcrest.Matchers;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
+/*
+ * https://material-ui.com/components/material-icons
+ */
+
 public class MaterialIconTests extends TestsInit {
 
+    @BeforeTest
+    public void beforeTest() {
+        openSection("Material Icons");
+        materialIconPage.isOpened();
+    }
+
+    private void lastClickTextIs(String text) {
+        lastClick.is().text(String.format("Last click:%s", text));
+    }
+
+    private void lastHoverTextIs(String text) {
+        lastHover.is().text(String.format("Last hover:%s", text));
+    }
+
     @Test
-    public void defaultMaterialIconTest() {
-        displayDataDefaultMaterialIconPage.open();
-        materialIconFrame.materialIcon.is().visible();
+    public void sizeAndColorTest() {
+        defaultAlarm.isDisplayed();
```

Comment:
for isDisplayed checking we need to use constructions like defaultAlarm.is().displayed()
same for other checks